wordle-web/src/components/AppBar.vue

```diff
@@ -0,0 +1,170 @@
+            <v-card-text> Misplaced: Orange </v-card-text>
+            <v-card-text> Wrong: Red </v-card-text>
+          </v-card>
+        </v-list-item>
+
+        <v-list-item>
+          <v-card color="#424242">
+            <v-btn color="#6D4C41" @click="toggleCowboyMode">Cowboy</v-btn>
+            <v-card-text> Correct: Blue </v-card-text>
+            <v-card-text> Misplaced: Light-Orange </v-card-text>
+            <v-card-text> Wrong: Blue-Grey </v-card-text>
+          </v-card>
+        </v-list-item>
+
+        <v-list-item>
+          <v-card color="#424242">
+            <v-btn color="#01579B" @click="toggleAquaMode">Aqua</v-btn>
+            <v-card-text> Correct: Teal </v-card-text>
+            <v-card-text> Misplaced: Yellow </v-card-text>
+            <v-card-text> Wrong: Pink </v-card-text>
+          </v-card>
+        </v-list-item>
+      </v-list>
+    </v-card>
+  </v-dialog>
+
+  <v-navigation-drawer v-model="menu" location="right" temporary>
+    <v-card rounded="lg" class="mt-6" elevation="0">
+      <v-card-title id="card-title" elevation="6"> <v-icon>mdi-menu</v-icon> Menu </v-card-title>
+
+      <v-list density="compact">
+        <br />
+
+        <v-list-item>
+          <v-btn to="/">Home</v-btn>
+        </v-list-item>
+
+        <v-list-item>
+          <v-btn to="/about">About</v-btn>
+        </v-list-item>
+
+        <v-list-item>
+          <v-btn to="/wordle">Play Game</v-btn>
+        </v-list-item>
+      </v-list>
+    </v-card>
+  </v-navigation-drawer>
+</template>
+
+<script setup lang="ts">
+import { ref, watch, onMounted } from 'vue'
+import { useTheme } from 'vuetify/lib/framework.mjs'
+import router from '@/router'
+import nightInKyoto from '@/assets/music/nightInKyoto.mp3'
+
+const theme = useTheme()
+let menu = ref(false)
+let settings = ref(false)
+const isPaused = ref(true)
+const audio = ref(new Audio(nightInKyoto))
+
+// TODO: Possibly remove, as it's not applicable since the it's not possible to press the menu
+//       button and the settings button at the same time, due to using 'v-dialog'.
+watch(menu, (val) => {
+  if (val) {
+    settings.value = false
+  }
+})
+
+watch(settings, (val) => {
+  if (val) {
+    menu.value = false
+  }
+})
+
+onMounted(() => {
+  audio.value.loop = true
+  audio.value.volume = 0.2
+})
+
+function toggleMusic() {
+  if (isPaused.value) {
+    audio.value.play()
+  } else {
+    audio.value.pause()
+  }
+  isPaused.value = !isPaused.value
+}
+
+function toggleDarkMode() {
+  theme.global.name.value = 'dark'
+  localStorage.setItem('theme', 'dark')
+}
+function toggleLightMode() {
+  theme.global.name.value = 'light'
+  localStorage.setItem('theme', 'light')
+}
+function toggleCowboyMode() {
+  theme.global.name.value = 'cowboy'
+  localStorage.setItem('theme', 'cowboy')
+}
+function toggleAquaMode() {
+  theme.global.name.value = 'aqua'
+  localStorage.setItem('theme', 'aqua')
+}
+</script>
+
+<style scoped>
```

Comment:
You can acheive alot of this without the css class, using padding and margin classes and the height/width attributes. Vuetify also has font classes under their typography docs